fix: error message for entering less than 4 characters for username

priyanshuone6 opened the pull request:
### Description
Fixed Error message for entering less than 4 characters for username under the update user profile and listed the error in the code section

Fixes #603 

### Type of Change:

<!--- **Delete irrelevant options.** --->

- Code


**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?

<!--Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test. -->
![Screenshot (32)](https://user-images.githubusercontent.com/64051212/93883566-5d46c300-fcff-11ea-8bde-b480fe7927ed.png)


### Checklist:

<!-- **Delete irrelevant options.** -->

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials


**Code/Quality Assurance Only**

- [x] My changes generate no new warnings

priyanshuone6 committed:
Merge pull request #1 from anitab-org/develop

Updated fork

priyanshuone6 committed:
fix error message under responses

priyanshuone6 committed:
fix error message under responses

SanketDG commented on app/api/resources/user.py:
```diff
@@ -129,7 +129,14 @@ def get(cls):
+         "Invalid input.",
+         messages.USERNAME_FIELD_IS_EMPTY
+    ),
+)
```
Prettifying this makes this more unreadable :)

let's just go with the orignal style.

priyanshuone6 committed:
change code style

priyanshuone6 commented:
@SanketDG I have changed the style, please review it and let me know

vj-codes reviewed (changes requested):
@priyanshuone6 can you please update the `how has this been tested part` .
I see the changes in HTTP.status Bad_request , but the response on top still shows longer than 4 and shorter than 26.
Maybe upload 2 screenshots one containing dummy data and second of responses.What say?

priyanshuone6 commented:
> @priyanshuone6 can you please update the `how has this been tested part` .
> I see the changes in HTTP.status Bad_request , but the response on top still shows longer than 4 and shorter than 26.
> Maybe upload 2 screenshots one containing dummy data and second of responses.What say?

@vj-codes I just added `messages.USERNAME_FIELD_IS_EMPTY` in `users_ns.response` and changed nothing except it.
Please have a look at the screenshots

![Screenshot (41)](https://user-images.githubusercontent.com/64051212/94388526-f72cc680-016a-11eb-86ce-588e2077dd50.png)
![Screenshot (42)](https://user-images.githubusercontent.com/64051212/94388528-f8f68a00-016a-11eb-9d4a-ba3899474e03.png)
![Screenshot (43)](https://user-images.githubusercontent.com/64051212/94388530-f98f2080-016a-11eb-835a-861f15aabe32.png)

vj-codes reviewed (changes requested):
@priyanshuone6 hey please solve the merge conflicts.Thanks!

priyanshuone6 committed:
Merge branch 'develop' into username-error

Aaishpra reviewed (changes requested):
@priyanshuone6 can you remove the merge conflicts(as suggested above) and run black app/api/resources/user.py to reformat the code according to black standards so that the failing test can pass.

priyanshuone6 committed:
Merge branch 'develop' into username-error

priyanshuone6 committed:
fix error message under responses

epicadk commented:
The tests are failing can you look into that @priyanshuone6

vj-codes commented:
@priyanshuone6 you've closed this PR , let us know if you would like to continue on this issue orthwise it'll be marked as available for other contributors.

priyanshuone6 commented:
This PR is open for about 5 months, due to which there are merge conflicts arising and I have solved them twice and no one's responding so I closed the PR.
You can mark the issue as open.
